.github/workflows/release.yml

```diff
@@ -0,0 +1,23 @@
+name: "release"
+on:
+  push:
+    tags:
+      - 'v*'
+jobs:
+  luarocks-upload:
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v3
+      - name: LuaRocks Upload
+        uses: nvim-neorocks/luarocks-tag-release@v1.0.2
+        env:
+          LUAROCKS_API_KEY: ${{ secrets.LUAROCKS_API_KEY }}
+        with:
```

Comment:
Seems like this is already part of rockspec. Does it need to be repeated here as well?

Reply:
The scm-1.rockspec isn't part of the workflow (and isn't necessary for it).
I added it because it can be used for local installs or to publish a `dev` version, which tells luarocks to build the package using the latest git source (see the `source.url`).

I guess it's more useful for plugins that have dependencies.